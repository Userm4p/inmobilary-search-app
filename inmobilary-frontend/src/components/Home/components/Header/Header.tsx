import React from 'react';
import { SearchBar } from '../SearchBar/SearchBar';
import { FiltersModal } from '../FiltersModal/FiltersModal';
import { Logo } from '@/components/Logo/Logo';

export const Header = () => {
  return (
    <div className="flex flex-col lg:flex-row items-center px-4 py-4 gap-4 color-black w-full">
      <Logo />
      <SearchBar />
      <FiltersModal>
        <span className="cursor-pointer text-wrap-nowrap text-white">Advance Search</span>
      </FiltersModal>
    </div>
  );
};
