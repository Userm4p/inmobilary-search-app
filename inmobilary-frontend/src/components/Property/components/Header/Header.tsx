import Link from 'next/link';
import React from 'react';
export const Header = () => {
  return (
    <div className="flex flex-row items-center px-4 py-4 gap-4 color-black w-full">
      <Link href={"/"} className="flex flex-row items-center gap-2">
        <h2 className="whitespace-nowrap font-bold text-shadow-lg text-white">Real Estate App</h2>
      </Link>
    </div>
  );
};
