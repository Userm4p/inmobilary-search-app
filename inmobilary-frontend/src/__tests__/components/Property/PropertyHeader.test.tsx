import { render } from '@testing-library/react';
import { Header } from '@/components/Property/components/Header/Header';

describe('Property Header', () => {
  it('should render header component', () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
