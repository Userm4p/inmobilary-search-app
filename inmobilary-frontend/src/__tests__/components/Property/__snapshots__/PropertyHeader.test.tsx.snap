// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Property Header should render header component 1`] = `
<div>
  <div
    class="flex flex-row items-center px-4 py-4 gap-4 color-black w-full"
  >
    <a
      class="flex flex-row items-center gap-2"
      href="/"
    >
      <h2
        class="whitespace-nowrap font-bold text-shadow-lg text-white border-4 border-blue-500 rounded-lg p-2"
      >
        Real Estate App
      </h2>
    </a>
  </div>
</div>
`;
